crates/admin/src/cluster_controller/service.rs

```diff
@@ -587,7 +587,7 @@ impl<T: TransportConnect> Service<T> {
                 info!(
                     ?log_id,
                     trim_point_inclusive = ?trim_point,
-                    "Manual trim log command received");
+                    "Trim log command received");
```

Comment:
I think this is slightly more accurate; it might have been issued by automation.